test/controllers/FinancialTransactionsControllerSpec.scala

```diff
@@ -184,10 +184,6 @@ class FinancialTransactionsControllerSpec extends SpecBase
             "return a status of 200 (OK)" in {
               status(result) shouldBe Status.OK
             }
-
-            "return a json body with the financial transaction information" in {
```

Comment:
Similar to above comment in IT tests, I think it makes more sense to keep this in but just change it to check for the current temporary result. You could also add a TODO if you think it will make it clearer, but either way once the next task is done it will start to fail. This ensures it's not forgotten.